docs/docs/pricing/README.md

```diff
@@ -9,137 +9,150 @@ We believe anyone should be able to run simple, low-volume workflows at no cost.
 
 To support these goals, **Pipedream offers a [generous free tier](#developer-tier)**. You can run sources and workflows for free within the limits of the free tier. If you hit these limits, you can upgrade to one of our [paid tiers](#professional-tier).
 
-Read more about our plans and pricing options below.
+[Read more about our plans and pricing here](https://pipedream.com/pricing).
 
-[[toc]]
+## Free Tier
 
-## Developer Tier
+You can run up to 3 workflows and add up to 3 connected accounts for up to 100 credits a day for free, as long as they operate under [Pipedream platform limits](/limits/).
 
-**The Developer tier is free**. You can run any number of sources and workflows, as long as they operate under [Pipedream platform limits](/limits/).
+**To lift the 100 credit limit and increase the number of workflows and connected accounts [upgrade to a paid tier](https://pipedream.com/pricing)**.
 
-**To run any number of invocations, for any amount of time, [upgrade to the Professional tier](https://pipedream.com/pricing)**.
+### Free Tier Connected Accounts
 
-### Developer Tier Support options
+Free Tier accounts can connect up to 3 different service accounts like Twitter, Discord, Google Sheets, or any of the thousands of available connections.
+
+### Free Tier Workflow Limitations
+
+Free Tier account workflows are limited to using a maximum of 100 credits per day and a maximum of 3 active workflows.
+
+Additionally, the event history for Free Tier workflows is limited to only the past 100 events.
+
+Upgrading to a paid tier will increase this event history tracking substantially.
+
+### Free Tier Polling Interval Limitations
```

Comment:
We actually aren't implementing polling frequency limits in the initial release, so I'd just remove this section entirely for now (and we'll update the pricing page when we do ship it).